content/html/concepts/attributes/terms/title/title.md

```diff
@@ -0,0 +1,33 @@
+**Title**
+
+**HTML global attribute**
+
+In the browser the ‘title’ attribute displays additional information about the HTML element in the form of a mouse over tooltip.
+
+**Syntax**
+
+The title attribute uses the *name=”value”* pair syntax.
+
+\<start tag **title=”value”**\>elements content\</end of tag\>
+
+For empty elements (without a closing tag):
+
+\<start tag **title=”value”**\>
+
+**Example**
+
+**![](media/9aee4aceea20998f4e0ffbc691e15ad4.png)**
+
+**Best Practice**
+
+Best practice suggests that the ‘title’ attribute be written in lower case letters, whilst this is not strictly necessary in HTML, other document types like XHTML demand lowercase.
+
+Further the value of the attribute should be surrounded by double quotation marks. If the value itself contains double quotation marks single quotation marks should be used. Multi word title values will not render correctly if quotation marks are omitted.
+
+**Additional Information**
```

Comment:
This entire section can be omitted as it's covered in the parent attribute section. A link to it will also be provided by the Docs navigation, as is the last line, which appears automatically after each Docs entry.